### PR TITLE
APP-3060 Documentation revamping, getting started

### PR DESCRIPTION
**Pull request by symphony-thibault**
Revamped `README.md`, Getting Started guide...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,33 +9,29 @@ The Symphony Java BDK helps you to create Bots and Applications on top of the [S
...
+Be sure that your `JAVA_HOME` environment variable points to the `jdk1.8+` folder extracted from the JDK download.
 
+### Get the source
 ```shell script
-# build the legacy modules
-./mvnw clean install
-
-# skip building legacy modules
-./mvnw clean install -P -legacy
+git clone git@github.com:SymphonyPlatformSolutions/symphony-api-client-java.git
```
Comment (symphony-youri) [inline README.md:23]: Is that necessary (because Github is already giving you that)?

File: README.md [hunk 2]
```diff
@@ -9,33 +9,29 @@ The Symphony Java BDK helps you to create Bots and Applications on top of the [S
 
 Documentation about BDK features and usage is available under [docs](./docs/index.md) folder.
 
-## How to Build
+## Build from Source
+The Symphony BDK uses a Maven [build](https://maven.apache.org/) build. The instructions below use the [Maven Wrapper](https://github.com/takari/maven-wrapper)
+from the root of the source treen. The wrapper script serves as a cross-platform, self-contained bootstrap mechanism for
```
Comment (symphony-youri) [inline README.md:14]: treen -> tree

File: README.md [hunk 3]
```diff
@@ -9,33 +9,29 @@ The Symphony Java BDK helps you to create Bots and Applications on top of the [S
 
 Documentation about BDK features and usage is available under [docs](./docs/index.md) folder.
 
-## How to Build
+## Build from Source
+The Symphony BDK uses a Maven [build](https://maven.apache.org/) build. The instructions below use the [Maven Wrapper](https://github.com/takari/maven-wrapper)
```
Comment (symphony-youri) [inline README.md:13]: build is written twice

File: README.md (hunk 1 above)
Comment (symphony-thibault) [reply README.md]: Yep, removed

File: README.md (hunk 2 above)
Comment (symphony-thibault) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (symphony-thibault) [reply README.md]: Fixed 👍

**Pull request merged**